qmi/instruments/wl_photonics/wltf_n.py

```diff
@@ -0,0 +1,334 @@
+        resp = self._read()
+        words = resp.rstrip().split("\r\n")
+        if len(words) != 3:
+            raise QMI_InstrumentException("Unexpected response to dev?, got {!r}".format(resp))
+
+        model = words[0].split(":")[0]
+        serial = search(r"SN\([\d]+\)", words[0])[0].strip("SN()")  # type: ignore
+        version = search(r"MD\([ \d-]+\)", words[0])[0].strip("MD()")  # type: ignore
+        # Update the wavelength and step ranges on-the-go
+        wl_range = words[1].split(":")[1].split("nm")[0].strip().split("~")
+        self._wavelength_range.min = float(wl_range[0])
+        self._wavelength_range.max = float(wl_range[1])
+        step_range = words[1].split(":")[2].rstrip(f"){self.response_terminator}").lstrip().split("~")
+        self._steps_range.max = int(step_range[0])  # NOTE: The range is inverse to the wavelength:
+        self._steps_range.min = int(step_range[1])  # Minimum step value is maximum wavelength and vice versa.
+
+        return QMI_InstrumentIdentification(vendor="WL Photonics", model=model, serial=serial, version=version)
+
+    @rpc_method
+    def set_center_wavelength(self, wavelength: float) -> None:
+        """Set the center wavelength in nanometers.
+
+        Parameters:
+            wavelength: The target wavelength in nanometers.
+
+        Raises:
+            QMI_InstrumentException: If the wavelength is not in the instrument range.
+        """
+        unit = "nm"
+        decimals = 3
+        if wavelength < self._wavelength_range.min or wavelength > self._wavelength_range.max:
+            raise ValueError(
+                f"Wavelength {wavelength}{unit} out of instrument range "
+                f"({self._wavelength_range.min}{unit} - {self._wavelength_range.max}{unit})"
+            )
+
+        cmd = f"wl{wavelength:.{decimals}f}"
+        self._write(cmd)
+        self._clear_ok(cmd)
+
+    @rpc_method
+    def get_center_wavelength(self) -> float:
+        """Get the center wavelength. Default unit is nanometers.
+
+        Returns:
+            wavelength: The instrument wavelength in nanometers.
+        """
+        wavelength = self._ask_float("wl?")
+        return wavelength
+
+    @rpc_method
+    def get_minimum_wavelength(self) -> float:
+        """Get the minimum wavelength.
+
+        Returns:
+            self._wavelength_range.min: The instrument wavelength minimum in nanometers.
+        """
+        return self._wavelength_range.min
+
+    @rpc_method
+    def get_maximum_wavelength(self) -> float:
+        """Get the maximum wavelength.
+
+        Returns:
+            self._wavelength_range.max: The instrument wavelength maximum in nanometers.
+        """
+        return self._wavelength_range.max
+
+    @rpc_method
+    def reverse_motor(self, steps: int) -> None:
+        """Reverse the motor by input steps. This can be used to fine-tune the center wavelength.
+
+        Parameters:
+            steps: How many steps the motor should reverse.
```

Comment:
"Number of steps the motor should reverse" sounds better to me. What do you think? Applies to the next method `forward_motor` as well

Reply:
I agree, that sounds better. Changing description.